data/priyanshugarg1010.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "Priyanhu Garg",
+  "displayStatsPublic": "true",
+  "type": "personal",
+  "bio": "I am a **fullstack developer** | Open Source",
+  "tags": ["Javascript", "NodeJS", "ReactJS", "Open Source"],
+  "avatar": "https://github.com/priyanshugarg1010.png",
```

the following is no longer needed:
```suggestion
  "bio": "I am a **fullstack developer** | Open Source",
  "tags": ["Javascript", "NodeJS", "ReactJS", "Open Source"],
```